src/css/SkipTo.template.css

```diff
@@ -122,7 +122,7 @@
 .skip-to [role="menuitem"].skip-to-nesting-level-3 .label { grid-column: 4 / 8; }
 
 .skip-to [role="menuitem"].no-items .label,
-.skip-to [role="menuitem"].action .label {
+.skip-to [role="menuitem"].skip-to-action .label {
```

Comment:
you cant just rename these things in a patch release. you are potentially breaking the markup of anyone who is using there own css. unless you are saying that the current class doesnt work